cmd/crl-storer/main.go

```diff
@@ -37,15 +37,17 @@ type Config struct {
 		// service, or to point to a fake service for testing. It should be left
 		// blank by default.
 		S3Endpoint string
-		// S3Region is the AWS Region (e.g. us-west-1) that uploads should go to.
-		S3Region string
 		// S3Bucket is the AWS Bucket that uploads should go to. Must be created
 		// (and have appropriate permissions set) beforehand.
 		S3Bucket string
-		// S3CredsFile is the path to a file on disk containing AWS credentials.
+		// AWSConfigFile is the path to a file on disk containing an AWS config.
+		// The format of the configuration file is specified at
+		// https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html.
+		AWSConfigFile string
+		// AWSCredsFile is the path to a file on disk containing AWS credentials.
 		// The format of the credentials file is specified at
 		// https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html.
-		S3CredsFile string
+		AWSCredsFile string
```

Comment:
Given that the AWS Config file is a superset of the AWS Credentials file we can probably deprecate this right in this commit by copying the value of `AWSCredsFile` to `AWSConfigFile`. Make sure you add a TODO and issue to pull this from the config altogether.

Reply:
I figured we'd still want to keep these separate, so that the ConfigFile could be provisioned onto the host like any other config file, while the CredsFile could be provisioned like our other secrets.

Reply:
Ah, I see! You may want to include a note for SRE concerning the precedence behavior described in the SDK docs:

>If duplicate profiles are specified within a same credentials/config file, the profile properties specified in the latter profile takes precedence.
>
>If duplicate profiles are specified across either multiple credentials files or across multiple config files, the profile properties are resolved as per the order of file input to the config.LoadOptions. The profile properties in the latter files take precedence.
>
>If a profile exists in both credentials file and config file, the credentials file properties take precedence.